library-extensions-expandable/src/main/java/com/mikepenz/fastadapter/expandable/ExpandableExtension.kt

```diff
@@ -22,13 +24,37 @@ fun <Item : GenericItem> FastAdapter<Item>.getExpandableExtension(): ExpandableE
  * Extension method to retrieve or create the ExpandableExtension from the current FastAdapter
  * This will return a non null variant and fail
  */
-fun <Item : GenericItem> FastAdapter<Item>.expandableExtension(block: ExpandableExtension<Item>.() -> Unit) {
+inline fun <Item : GenericItem> FastAdapter<Item>.expandableExtension(block: ExpandableExtension<Item>.() -> Unit) {
     getExpandableExtension().apply(block)
 }
 
+/**
+ * internal helper function to check if an item is expanded.
+ */
+internal fun IItem<out RecyclerView.ViewHolder>?.isExpanded(): Boolean {
```

Comment:
Not sure if we should use a extension function or property-.

Reply:
But if the property would be on the `IItem` then internal would prevent it from being accessible in the expandable module

Reply:
It's possible to create extension properties not just functions.

Reply:
I have no strong opinion on this here. Feel free to adjust :)

Reply:
ok went ahead and changed it :D